torch/linalg/__init__.py

```diff
@@ -488,6 +488,90 @@
     >>> tensor([ 11.6734+0.j, 105.1037+0.j,  10.1978+0.j])
 """)
 
+pinv = _add_docstr(_linalg.linalg_pinv, r"""
+linalg.pinv(input, rcond=1e-15, hermitian=False) -> Tensor
+
+Computes the pseudo-inverse (also known as the Moore-Penrose inverse) of a matrix :attr:`input`,
+or of each matrix in a batched :attr:`input`.
+The pseudo-inverse is computed using singular value decomposition (see :func:`torch.linalg.svd`) by default.
```

Comment:
Unfortunately torch.linalg.svd doesn't exist yet. This can reference torch.svd for now.

Reply:
Sure I can change that.
I just thought that we should start using `torch.linalg` even if the references are not working for now.
> While we're not actually using the newer linear algebra functions internally we should still reference them here.
https://github.com/pytorch/pytorch/pull/48206#discussion_r526637718

Reply:
It is nice to dream... but we should probably keep all the references in the docs working ;)